test/test_diff_tar.py

```diff
@@ -0,0 +1,106 @@
+import os
```

Comment:
The rest of the test suite uses pytest.  Please adjust this to confom to that style. Thanks!

Reply:
Hmm, pytest supports unittest out of the box, see https://docs.pytest.org/en/latest/unittest.html .  I can run `pytest test_diff_tar.py` just fine, so I'm not sure what to adjust here.

Reply:
While they execute just fine, it is a totally different style which makes maintenance harder. Please change the tests here to use pytest-style with fixtures.